packages/react/runtime/src/snapshot/ref.ts

```diff
@@ -137,12 +137,12 @@ function markRefToRemove(sign: string, ref: unknown): void {
 }
 
 export {
-  updateRef,
-  takeGlobalRefPatchMap,
-  updateBackgroundRefs,
-  unref,
-  transformRef,
   globalRefsToRemove,
   globalRefsToSet,
   markRefToRemove,
+  takeGlobalRefPatchMap,
```

Comment:
@colinaaa Does our formatter guarantee a stable and consistent output?

Reply:
Yes, it does. It seems this was modified by the ESLint auto-fix (we're using `eslint-plugin-import`, which organizes the import/export order).